docs/91_version_history.rst

```diff
@@ -25,6 +25,152 @@ Version History
+       - The asset graph in the graphical user interface woudl sometimes not resize to fit the screen correctly, this has now been resolved.
+       - The “Asset & Readings” option in the graphical user interface was initially slow to respond, this has now been improved.
+       - The pagination of audit logs has bene improved when multiple sources are displayed.
+       - The counts in the user interface for notifications have been corrected.
+       - Asset data graphs are not able to handle correctly the transition between one day and the next. This is now resolved.
+
+
+- **Plugins**
+
+    - New Features:
+
+       - The existing set of OMF north plugins have been rationalised and replaced by a single OMF north plugin that is able to support the connector rely, PI Web API, EDS and OCS.
+       - A new notification rule plugin has been added that will trigger periodically if data is observed for a given asset. The trigger interval is defined in seconds. The plugin is primarily designed to be used with conditional forwarding as a means to sample data, although other uses might include a watchdog implementation for particular data streams.
+       - When a Modbus TCP connection is closed by the remote end we fail to read a value, we then reconnect and move on to read the next value. On device with short timeout values, smaller than the poll interval, we fail the same reading every time and never get a value for that reading. The behaviour has been modified to allow us to retry reading the original value after re-establishing the connection.
+       - The OMF north plugin has been updated to support the released version of the OSIsoft EDS product as a destination for data.
+       - New functionality has been added to the north data to PI plugin when using PI Web API that allows the location in the PI Server AF hierarchy to be defined. A default location can be set and an override based on the asset name or metadata within the reading. The data may also be placed in multiple locations within the AF hierarchy.
+       - A new notification delivery plugin has been added that allows a north task to be triggered to send data for a period of time either side of the notification trigger event. This allows conditional forwarding of large amounts of data when a trigger event occurs.
+       - The asset notification delivery plugin has been updated to allow creation of new assets both for notifications that are triggered and/or cleared.
+       - The rate filter now allows the termination of sending full rate data either by use of an expression or by specifying a time in milliseconds.
+       - A new downsampling filter has been written that allows the rate of data to be reduced using one of a number of different algorithms.
+       - A new plugin that support the Digiducer 333D01 vibration sensor has been added.
+       - A new simple Python filter has been added that calculates an exponential moving average,
```

Comment:
`filter-downsample`, `south-digiducer`, `filter-simple-python` plugins are not applicable for fledge release